### ISSUE TITLE
Not-CSS: Samsung s10+ Firefox: Missing title in Personal Details drop down menu

### ISSUE THREAD

**Issue by KateChuen** [SAP/spartacus#3490]
**Environment Details**
Spartacus: Release 1.0
Browser: Firefox
Device: samsung s10+
Happens on: https://spartacus.c39j2-walkersde1-d4-public.model-t.cc.commerce.ondemand.com/electronics-spa/de/USD/

**Steps to Reproduce**
Go to MyAccount -> Personal Details
Click on the drop down menu for title.

**Actual Result**
The arrow for the drop down menu is too much on the right.
![abcc3f83-e38a-4a69-9c38-3c322bd2daf1](https://user-images.githubusercontent.com/45409515/60473035-58e54480-9c39-11e9-8e04-3cbd3711af9d.JPG)

Then when the menu is open, the first radio button doesn't have a title name to it.
![9adebfd9-cdd5-4f4f-8c4c-0555559ca96f](https://user-images.githubusercontent.com/45409515/60473073-761a1300-9c39-11e9-9121-086f98be3b28.JPG)

**Comment by morganm58** [comment]
The dropdown right now is done using `select` in this case should be a `ng-select.` That would fix the problem and make the select drop downs more consistent.

**Comment by developpeurweb** [comment]
The difference is more evident on Safari.

![Screen Shot 2020-05-29 at 1 02 17 PM](https://user-images.githubusercontent.com/15113219/83286407-dac55100-a1ad-11ea-9254-4ee2eb58a596.png)

**Comment by saad-mo** [comment]
The same issue is happening on the `Create An Account` page. PR for this issue will cover both areas. 

<img width="567" alt="Screen Shot 2020-08-12 at 12 54 37 PM" src="https://user-images.githubusercontent.com/15215400/90049889-1777f300-dc9b-11ea-918e-2cc5bd96d689.png">

**Comment by saad-mo** [comment]
As discussed in the post-scrum (10/13/2020), this issue will be addressed as part of the a11y/screen-readers initiative

**Comment by FollowTheFlo** [comment]
bug found:
1 -> login, go personal details, change title (Mr,Miss,Ms...) save, logout.
2 -> login again, a blank line is added to selection
Do steps 1-2 again, new blank line gets concatenated
![image](https://user-images.githubusercontent.com/30024415/157297755-ce65fc2e-ad6c-432a-97ed-5013a600aa14.png)
